content/docs/reference/install.md

```diff
@@ -16,15 +16,15 @@ This page contains detailed instructions for [installing Pulumi](#install-pulumi
 
 {{< get-started-note >}}
 
-## Available versions
-
-The current stable version of Pulumi is **{{< latest-version >}}**. For a full history of prior versions, including
-release notes and direct download links, visit the <a href="{{< relref "/docs/reference/changelog" >}}">Changelog</a> page.
-Pulumi is packaged as a tarball for Linux and macOS, and as a zip archive for Windows.
-
-
 ## Install Pulumi
 
+<div class="note note-warning" role="alert">
```

Comment:
@justinvp Thought I'd address https://github.com/pulumi/docs/issues/1473 as well while we're working on the installation page. Let me know your thoughts.

Reply:
If it were me, I would do this in its own PR. It's also not clear to me that we want this note here at the top of the page as this is talking about a version check before even discussing installing Pulumi. Perhaps we need another "Version Checks" section (or better name) after the install instructions sections, which could discuss how Pulumi will emit a warning if the current running version is out-of-date, and in that section add a note about the environment variable. We should also consider finding a place on another page where we could document all environment variables used by the CLI.

Reply:
Sounds good. Let's revisit the environment variable doc work at next week's sync meeting. I retained the additional styles but removed the note -- will work on #1473 in a separate PR.